### DISCUSSION TITLE
How to set CursorLineNr of nvim-tree?

### DISCUSSION THREAD

**Discussion by longguzzz** [nvim-tree/nvim-tree.lua#1611] [Q&A] [answered]
`NvimTreeCursorLine` is in nvim-tree-lua.txt, but I don't find something like NvimTreeCursorLineNr. When I try setting NvimTreeCursorLineNr, nothing happens. Is there any way to set NvimTreeCursorLineNr?

**Comment by alex-courtis** [comment, marked as answer]
That doesn't exist however it's a great idea and can be easily achieved.

Please raise a feature request and a PR.

**Reply by longguzzz** [reply]
OK. https://github.com/kyazdani42/nvim-tree.lua/pull/1616
It works in my nvim.
![image](https://user-images.githubusercontent.com/96779663/193395670-0087f509-d128-4aa5-b527-415c55f28915.png)
